### PR TITLE
Api project

### PR DESCRIPTION
**Pull request by RinaSol**
Adding my three files for Review.

### CODE REVIEW

File: GET.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ItemOrdered | string | Item ordered by the client
+type | string | Type of meal ordered
+Cost | Float | Cost of the meal
+Item2 | string | Second item ordered
+
+
+
+# GET Request Curl  
+  
+curl -X GET "http://URL/tableNo?id=99"
```
Comment (Laura-Novich-OBW) [inline GET.md:29]:
```suggestion
```curl -X GET "http://URL/tableNo?id=99"```
```

File: POST.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+|Patty type|Beef, chicken, lamb or vegan|Choice of the patty|Beef|string|R|  
+|Burger size|160g, 220g or 300gm|Weight of the Burger|220g|Integer|R|
+|Bread type|White, whole wheat, multigrain or gluten free|Type of bread|white|string|R|
+|Cook level|medium, rare or well done|how much the patty needs to be cooked|well done|string|R|
+|Condiments|ketchup, mustard, Mayonnaise, barbeque sauce or hot sauce|what condiments the client would like|ketchup|string|O|
+|Toppings|fried onions, fried mushrooms, fried egg or none|whether the client would like a topping on his burger|none|string|O|
+|Sides|french fries, onion rings, salad, green beans, none|which sides would the client like|french fries|string|O|
+|Side Size|medium, large and extra large|The size of the side dish|medium|string|O|
+|Drinks|cola regular or zero, sprite, fanta, water, iced tea, soda, apple juice and grape juice|choice of the drink the client would like with his meal|cola regular|string|R|
+|Drink size|medium, large or extra large|which size would the client like for his drink that is from the fountain and not a bottled drink|medium|string|O|
```
Comment (Laura-Novich-OBW) [inline POST.md:19]:
```suggestion
|Drink size|medium, large, or extra large|fountain drink size|medium|string|O|
```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
Please make the following changes

**Comment by RinaSol** [comment]
Both GET.md and POST.md updated as requested

**Comment by RinaSol** [comment]
Both GET.md and POST.md updated as requested

File: GET.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Method | syntax
+----- | ----------
+GET | base_url/endpoint/etc.
+
+# GET Request  
+NAME  | Type  | Description  
+----- | ----| -----  
+OrderNum  | string | Number exclusive for ordering the meal. The table number or takeout order.
+timestamp | timestamp | The time the meal was ordered.
```
[the remark this answers is not in the document]
Comment (RinaSol) [reply GET.md]: Update done

File: GET.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Item1 | string | First item ordered
+ItemOrdered | string | Item ordered by the client
+type | string | Type of meal ordered
+Cost | Float | Cost of the meal
```
[the remark this answers is not in the document]
Comment (RinaSol) [reply GET.md]: Update completed

File: GET.md (hunk 1 above)
Comment (RinaSol) [reply GET.md]: Update completed

File: POST.md (hunk 2 above)
Comment (RinaSol) [reply POST.md]: Update to POST file completed.

**Pull request merged**